packages/engine/src/gltf/GLTFComponent.tsx

```diff
@@ -248,13 +256,23 @@ const ResourceReactor = (props: { documentID: string; entity: Entity }) => {
   const resourceQuery = useQuery([SourceComponent, ResourcePendingComponent])
   const gltfDocumentState = useMutableState(GLTFDocumentState)
   const sourceEntities = useHookstate(SourceComponent.entitiesBySourceState[props.documentID])
+  const childMeshEntities = useChildrenWithComponents(props.entity, [MeshComponent])
```

Comment:
are all these hooks still needed?

Reply:
nope, missed removing them when moving to external file